ballsdex/packages/admin/cog.py

```diff
@@ -1148,15 +1148,17 @@ async def balls_reset(
                 "The user you gave does not exist.", ephemeral=True
             )
             return
-        if percentage and not 0 < percentage < 100:
+        if percentage and not 0 < percentage <= 101:
```

Comment:
You're allowing `101` as a value there
```suggestion
        if percentage and not 0 < percentage <= 100:
```